views/partials/head.ejs

```diff
@@ -109,7 +109,7 @@
 
 	}
 
-	function rewriteUrl(lang, reload = false){
+	function rewriteUrl(lang = 'en', reload = false){
```

Comment:
how about
```suggestion
	function rewriteUrl(lang, reload = false){
	    if (!lang) {
	        lang = 'en';
	    }
```
this way it also works if someone explicitly passes a null (ie., `rewriteUrl(null, false)`

Reply:
okay, I'll do it on my side then. The change you're doing here doesn't change the behavior.